Subscriber/ApplePayDirectSubscriber.php

```diff
@@ -115,6 +194,55 @@ public function onFrontendCheckoutPostDispatch(Enlight_Event_EventArgs $args)
         $view->assign('sPayments', $sPayments);
     }
 
+    /**
+     * @param int $customerId
+     * @param int $paymentId
+     * @throws \Doctrine\ORM\ORMException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     */
+    private function updateCustomerDefaultPaymentMethod(int $customerId, int $paymentId) {
```

Comment:
could you auto-format that file
the { and a few other things tells me it has not been done, just a small thing, thank you

Reply:
Done

Reply:
csfixer implementation would solve this

Reply:
thats the reason why i'm currently adding it anyway in the meanwhile